### PR TITLE
[obs] further restrict NodePoolLoad to avoid false positives

### PR DESCRIPTION
**Pull request by kylos101**
## Description

Only trigger alerts when 2 or more nodes have high load average that is sustained over 1 for 60m.

## Related Issue(s)

Fixes WKS-301

## How to test

We don't actually have data for when [this](https://grafana.gitpod.io/explore?left=%7B%22datasource%22:%22P4169E866C3094E38%22,%22queries%22:%5B%7B%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22P4169E866C3094E38%22%7D,%22expr%22:%22sum%20by%28nodepool%29%20%28count%20by%28node,%20nodepool%29%20%28sum%20by%28node,%20nodepool%29%20%28nodepool:node_load1:normalized%7Bnodepool%3D~%5C%22.%2Aworkspace.%2A%5C%22,%20cluster%21~%5C%22ephemeral.%2A%5C%22%7D%29%20%3E%3D%201%29%29%20%3E%3D%202%22,%22format%22:%22time_series%22,%22intervalFactor%22:2,%22refId%22:%22A%22,%22interval%22:%22%22,%22editorMode%22:%22builder%22,%22range%22:true,%22instant%22:true,%22hide%22:false,%22legendFormat%22:%22__auto%22%7D,%7B%22datasource%22:%7B%22type%22:%22prometheus%22,%22uid%22:%22P4169E866C3094E38%22%7D,%22expr%22:%22sum%20by%28nodepool%29%20%28count%20by%28node,%20nodepool%29%20%28sum%20by%28node,%20nodepool%29%20%28nodepool:node_load1:normalized%7Bnodepool%3D~%5C%22.%2Aworkspace.%2A%5C%22,%20cluster%21~%5C%22ephemeral.%2A%5C%22%7D%29%20%3E%3D%202%29%29%20%3E%201%22,%22format%22:%22time_series%22,%22intervalFactor%22:2,%22refId%22:%22B%22,%22interval%22:%22%22,%22editorMode%22:%22builder%22,%22range%22:true,%22instant%22:true,%22hide%22:true,%22legendFormat%22:%22%7B%7Bnodepool%7D%7D%22%7D%5D,%22range%22:%7B%22from%22:%22now-30d%22,%22to%22:%22now%22%7D%7D&orgId=1) would have triggered, because that was before June 9.

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

**Comment by kylos101** [comment]
@selfcontained to help avoid future alerts this week, as you're on L1. After this merges, I'll make sure it's deployed in monitoring central.

**Comment by kylos101** [comment]
> @selfcontained to help avoid future alerts this week, as you're on L1. After this merges, I'll make sure it's deployed in monitoring central.

Note: no need for your approval, we just need from Workspace Team

**Review by aledbf — APPROVED** [review]

**Pull request merged**
